playwright/test-runner-jest.config.js

```diff
@@ -1,18 +1,9 @@
-const { TEST_ROOT, TEST_MATCH } = process.env;
-
-const roots = TEST_ROOT ? [TEST_ROOT] : undefined;
+const { getJestConfig } = require('@storybook/test-runner');
 
 module.exports = {
-  cacheDirectory: 'node_modules/.cache/storybook/test-runner',
-  rootDir: process.cwd(),
-  roots,
-  testMatch: [TEST_MATCH || '**/*.stories.[jt]s?(x)'],
-  transform: {
-    '^.+\\.stories\\.[jt]sx?$': '@storybook/test-runner/playwright/transform',
-    '^.+\\.[jt]sx?$': 'babel-jest',
-  },
-  preset: 'jest-playwright-preset',
-  globalSetup: '@storybook/test-runner/playwright/global-setup.js',
-  globalTeardown: '@storybook/test-runner/playwright/global-teardown.js',
-  testEnvironment: '@storybook/test-runner/playwright/custom-environment.js',
-};
+  // The default configuration comes from @storybook/test-runner
+  ...getJestConfig()
+  /** Add your own overrides below
```

Comment:
Nice! But is Jsdoc the right comment format (minor nit)?

Reply:
Was intentional to have the comment highlight the link so it won't go unnoticed (I stole the idea from [redwood's sb config page](https://redwoodjs.com/docs/storybook#configuring-the-server-with-storybookconfigjs))

![image](https://user-images.githubusercontent.com/1671563/153285879-6da770da-e5de-4218-805a-de7b18d7d34f.png)

Happy to remove it!